### DISCUSSION TITLE
headlessui for vue 2.0 Release

### DISCUSSION THREAD

**Discussion by Tchilly** [tailwindlabs/headlessui#3426] [Ideas]
Hey

Been using this repo for a while, and my question; "Are we going to see a 2.0 release of the Vue version?". I also use the react version and it's great, but my heart is with Vue, and would love to see a 2.0 soon.

Keep up the good work, and thank you.

**Comment by LucaRed** [comment]
It's been already discussed in #3183. It seems like we have to wait, as [Adam wrote on Twitter](https://x.com/adamwathan/status/1807502683189772362):

>Hey! No concrete plans to do Catalyst for Vue but not something I'm actually against, just hasn't earned a spot on the roadmap. Headless UI v2 for Vue I just don't have a good answer for — I think we should do it but our plate is already full for the rest of the year with Tailwind CSS v4.0 related projects so I just don't really know yet what we're going to do.

Honestly I don't mind waiting, I just hope the project won't be abandoned.

**Reply by Tchilly** [reply]
I have a license for tailwindui components, and one of the reasons were Vue and headlessUI playing very nicely together.  Since using the latest react version of headless, there are several improvements I would love to see "migrated" to the Vue version. I really hope it gets a bump :D

**Comment by MartinMa** [comment]
A blocker for a Vue 2.0 release might also be the new dependencies `@react-aria/focus` an `@react-aria/interactions` for which no Vue equivalent exists (at least to my knowledge).

**Reply by Tchilly** [reply]
https://vueuse.org/core/useFocus/

Is this not the "same"?
